scantools/capture/namedposes.py

```diff
@@ -0,0 +1,24 @@
+from pathlib import Path
+from .pose import Pose
+from ..utils.io import read_csv, write_csv
+
+
+class NamedPoses(dict):
```

Comment:
Isn't this redundant with `capture.proc.GlobalAlignment`?

Reply:
ok.. I can write here now. was looking at it with the wrong account. I changed it using GlobalAlignment.